src/taco-cli/cli/commands.json

```diff
@@ -444,6 +451,14 @@
             {
                 "name": "--target=TARGET", "description": "[CommandRunTargetDescription]"
             }
+            ,
+            {
+                 "name": "--livereload", "description": "[CommandRunLiveReloadDescription]"
```

Comment:
I see livereload/devicesync option handlers in run.ts only. You will have to make similar changes to emulate.ts as well.